.eslintrc

```diff
@@ -1,12 +1,14 @@
 {
     "env": {
-        "es6": true
+        "es6": true,
+        "node": true
```

Comment:
It's better to move this env variable to the `./build` folder

Reply:
`./build` has such settings, It should be moved into `./testing/renovation`

Reply:
done